### ISSUE TITLE
Extension issue

### ISSUE THREAD

**Issue by samdruster** [MicrosoftDocs/intellicode#279]
- Issue Type: `Bug`
- Extension Name: `vscodeintellicode`
- Extension Version: `1.2.12`
- OS Version: `Windows_NT x64 10.0.19042`
- VSCode version: `1.54.3`

:warning: We have written the needed data into your clipboard. Please paste! :warning:

**Comment by markw-t** [comment]
@samdruster thanks for the report - we need a little more detail of what you are seeing before we can make any progress with this issue. Please can you add the info below so we can investigate further? Thanks!:

1. What language are you trying to use IntelliCode with? (Python/JavaScript/TypeScript/Java/SQL)?
2. What happens when you try to type a method/property in that language for a common class - do you see IntelliCode "stars" in the autocomplete list?
3. Can you open up the Output pane (View menu.... Output), then select VS IntelliCode from the dropdown on the top right bar of that pane, and copy the contents of the pane here? It will often contain useful information to help with debugging.  It will look something like this (here for Python):

![image](https://user-images.githubusercontent.com/12836119/113321407-6a54f800-92c8-11eb-8bc3-4444ac209a06.png)

**Comment by ayim** [comment]
Hi @samdruster are you still experience issues here?
Feel free to open a new issue if you are, would love to hear from you.
